tests/components/androidtv/test_media_player.py

```diff
@@ -245,7 +235,8 @@ def _setup(hass, config):
     * ADB connection method: ADB server
 
     """
-    assert await _test_adb_shell_returns_none(hass, CONFIG_ANDROIDTV_ADB_SERVER)
+    assert await _test_adb_shell_returns_none(hass,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4678217)